packages/design-system/src/styles/components/_Dropdown.scss

```diff
@@ -58,16 +66,48 @@
   .ds-c-field--inverse + & {
     color: var(--color-base);
   }
+
+  // When dropdown is open (not `hidden`), add border radius to bottom of menu
+  &:not([hidden]) {
+    // We don't have a good way to create alpha versions of colors using our current hex value tokens
+    // Sass has color functions, but they can't be used in CSS custom properties because CSS custom properties are evaluated at runtime
+    // Hex values can't be used in native CSS color functions like `hsla()` or `rgba()`
+    // Creating a temporary variable until we discuss a better solution
+    --shadow-color: 0deg 0% 50%;
+    animation: fade-in var(--animation-speed-1) var(--animation-ease-in-out-expo);
```

Comment:
When @brentschneider and I talked last, I think we decided that a new bezier curve and animation speed needed to be created for this. I think @brentschneider was going to define what that curve looked like, but you may need to work together on it.